### ISSUE TITLE
Disable keyboard input

### ISSUE THREAD

**Issue by TheGordinho** [etra0/botw-freecam#10]
Set up a hotkey to disable keyboard input or automaticly disable it on alt-tab if possible

**Comment by etra0** [comment]
Well, seems like there's no easy way to quickly do this, in order to do a "proper" implementation I'd have to rewrite the entire input system. The quick way of doing this is a button to block keyboard input but doesn't seems to be a good solution since it'll require more interaction from the user.

This implementation will take me much more time than expected.

**Comment by TheGordinho** [comment]
Thanks for taking the time to look into it

**Comment by mistermark** [comment]
I understand this takes a lot of effort to fix, but I also am facing the same issue.

I often leave my game running while doing some other things on the side on the same PC, which makes Link move, or the cam move unintentionally because of inputs captured from outside the game.

**Comment by etra0** [comment]
@mistermark Yes, it is something I have in mind, but as I already said, it'll take a lot of time because I have to rewrite the entire input system. Right now the solution I see is hooking into `GetMessage` to at least receive the message of when the window is on focus, but this means I have to update `memory-rs` to be able to easily hook to functions outside of Cemu.
